server/api/events.js

```diff
@@ -5,7 +5,7 @@ module.exports = router
 router.get('/', async (req, res, next) => {
   try {
     const events = await Event.findAll({
-      where: {public: false},
+      where: {public: true},
```

Comment:
default value for Events is false right now... There is no way to set an event to "True" on the front end right now... So this should definitely be false still UNTIL we add in that logic... or else no events will show up